### PR TITLE
Merged changes from nf-core template

### PR DESCRIPTION
**Pull request by d4straub**
TEMPLATE branch had issues, was [reset](https://nf-co.re/developers/sync#fixing-a-broken-template-branch), the current automated PR for template update 12.1 was closed by this. This PR is a [manual synchronization](https://nf-co.re/developers/sync#manual-synchronisation) with the most recent template using `nf-core sync`.

### CODE REVIEW

**Comment by d4straub** [comment]
Obvious changes that need to be made include:
- conf/base.config rm line 22-30
- rm  conf/igenomes.config entirely
- nextflow.config rm lines 79, 85, 86

File: README.md [hunk 1]
```diff
@@ -38,25 +40,39 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
...
-
 ## Credits
 
-These scripts were originally written for use at the [Quantitative Biology Center (QBiC)](http://www.qbic.life) and [Microbial Ecology, Center for Applied Geosciences](http://www.uni-tuebingen.de/de/104325), part of Eberhard Karls Universität Tübingen (Germany) by Daniel Straub ([@d4straub](https://github.com/d4straub)) and Alexander Peltzer ([@apeltzer](https://github.com/apeltzer)).
+nf-core/ampliseq was originally written by Daniel Straub ([@d4straub](https://github.com/d4straub)) and Alexander Peltzer ([@apeltzer](https://github.com/apeltzer)) for use at the [Quantitative Biology Center (QBiC)](http://www.qbic.life) and [Microbial Ecology, Center for Applied Geosciences](http://www.uni-tuebingen.de/de/104325), part of Eberhard Karls Universität Tübingen (Germany).
+
+We thank the following people for their extensive assistance in the development
```
Comment (skrakau) [inline README.md:66]: I would comment this out for now, if anyone no one is listed

**Review by skrakau — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (d4straub) [reply README.md:66]: Contributers will be added in a coming PR.

**Pull request merged**